src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,18 @@
 import React from 'react';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import { Comment } from '../../types/Comment';
+
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <div className="CommentList">
+    {comments.map(comment => (
+      <CommentInfo key={comment.id} comment={comment} />
```

Comment:
Consider using a bunch of props here for a better understanding of what is needed for your info component directly